rogue-thi-app/pages/food.jsx

```diff
@@ -59,7 +60,15 @@ export default function Mensa () {
   const userKind = useUserKind()
   const router = useRouter()
   const { i18n, t } = useTranslation('food')
-  const currentLocale = i18n.languages[0]
+
+  let languageFood = ''
```

Comment:
Can be simplified to:

```js
const currentLocale = selectedLanguageFood && selectedLanguageFood !== 'default' ? selectedLanguageFood : i18n.languages[0]
```

Therefore, all the `languageFood` referenced should be changed to `currentLocale`